subprojects/language-native/build.gradle.kts

```diff
@@ -71,7 +71,9 @@ packageCycles {
     excludePatterns.add("org/gradle/language/nativeplatform/internal/**")
 }
 
-integTest.usesJavadocCodeSnippets.set(true)
+integTest {
+    usesJavadocCodeSnippets = true
+}
```

Comment:
🤔 One benefit of keeping the existing format is that way we can look at the PR and see how many lines were added/removed as a result.

Reply:
This is a workaround for https://youtrack.jetbrains.com/issue/KT-56221/, but I can also revert it. What would be the benefit in removed/added line count in this case?

Reply:
reverted